client/state/themes/test/actions.js

```diff
@@ -121,4 +125,91 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#requestActiveTheme', () => {
+		const successResponse = {
+			id: 'rebalance',
+			screenshot: 'https://i0.wp.com/s0.wp.com/wp-content/themes/pub/rebalance/screenshot.png?ssl=1',
+			cost: {
+				currency: 'USD',
+				number: 0,
+				display: ''
+			},
+			version: '1.1.1',
+			download_url: 'https://public-api.wordpress.com/rest/v1/themes/download/rebalance.zip',
+			trending_rank: 17,
+			popularity_rank: 183,
+			launch_date: '2016-05-13',
+			name: 'Rebalance',
+			description: 'Rebalance is a new spin on the classic ' +
+				'Imbalance 2 portfolio theme. It is a simple, modern' +
+				'theme for photographers, artists, and graphic designers' +
+				'looking to showcase their work.',
+			tags: [
+				'responsive-layout',
+				'one-column',
+				'two-columns',
+				'three-columns',
+				'custom-background',
+				'custom-colors',
+				'custom-menu',
+				'featured-images',
+				'featured-content-with-pages',
+				'theme-options',
+				'threaded-comments',
+				'translation-ready'
+			],
+			preview_url: 'https://budzanowski.wordpress.com/?theme=pub/rebalance&hide_banners=true'
+		};
+
+		const failureResponse = {
+			status: 404,
+			code: 'unknown_blog',
+			message: 'Unknown blog'
+		};
+
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.get( '/rest/v1.1/sites/2211667/themes/mine' )
+				.reply( 200, successResponse )
+				.get( '/rest/v1.1/sites/666/themes/mine' )
+				.reply( 404, failureResponse );
+		} );
+
+		it( 'should dispatch current theme request action when triggered', () => {
```

Comment:
Let's stick to _active theme_ for the nomenclature 😉  (same in the `it` statements below).